templates/workspaces/base/terraform/airlock/storage_accounts.tf

```diff
@@ -296,3 +336,44 @@ resource "azurerm_role_assignment" "api_sa_data_contributor" {
   role_definition_name = "Storage Blob Data Contributor"
   principal_id         = data.azurerm_user_assigned_identity.api_id.principal_id
 }
+
+resource "azurerm_storage_account_customer_managed_key" "sa_import_approved_encryption" {
```

Comment:
Have you considered using a for_each here?

Reply:
thanks, fixed